air/src/execution_step/execution_context/scalar_variables.rs

```diff
@@ -116,6 +120,16 @@ impl<'i> Scalars<'i> {
         self.canon_streams.set_value(name, value)
     }
 
+    /// Returns true if there was a previous value for the provided key on the same
+    /// fold block.
+    pub(crate) fn set_canon_map_value<'k: 'i>(
```

Comment:
In map context, value has special meaning.  I think this name is misleading.

Reply:
However it would contradict with the naming pattern used around for scalar and canon.